tests/unit/bitops.tcl

```diff
@@ -128,6 +128,15 @@ start_server {tags {"bitops"}} {
         }
     }
 
+    test {BITCOUNT with just start} {
+        set s "foobar"
+        r set s $s
+        assert_equal [r bitcount s 0] [count_bits "foobar"]
+        assert_equal [r bitcount s 1] [count_bits "oobar"]
+        assert_equal [r bitcount s -1] [count_bits "r"]
+        assert_equal [r bitcount s -2] [count_bits "ar"]
```

Comment:
Could you please add several special cases which include larger value of start, such as start = 10 or 50, Thanks

Reply:
I'll add some new test cases in near days. thxs

Reply:
Added more testcases; https://github.com/valkey-io/valkey/pull/118/commits/843c84a09142dad2ecf938e292053828befbe39e

1. Larger value of start as you said
2. Larger negative value of start.
3. Also added large negative value in `BITCOUNT with start, end`
4. fix [count_bits ""] -> 0, when start > end if it is in the range of string (line 147)

Reply:
note: you said large value as 10 or 50 for example, but the value 1000 is already used from others, so I also use 1000.

Reply:
@hwware gently ping, just wondered if you didn't receive above comments alarm.
i hope it didn't bother you.